projects/plugins/jetpack/modules/widget-visibility/widget-conditions.php

```diff
@@ -10,24 +10,24 @@ class Jetpack_Widget_Conditions {
 	static $passed_template_redirect = false;
 
 	public static function init() {
-		if ( is_admin() ) {
+		global $pagenow;
+
+		if ( is_customize_preview() || 'widgets.php' === $pagenow || 'themes.php' === $pagenow ||
```

Comment:
@jeherve, do we actually need to run this code in `themes.php`? Haven't found a reason yet.

Reply:
Maybe this was added to target `themes.php?page=gutenberg-widgets`, the new widget screen? But that may not be necessary here since we check for `/wp-json/wp/v2/widget-types` as well a bit downer.

Reply:
Ahh yeah makes sense! We will probably remove `themes.php` and rely only in the `/wp-json/wp/v2/widget-types` then.